### DISCUSSION TITLE
Osu Crashed and Clear .osu mapping file data while editing

### DISCUSSION THREAD

**Discussion by Mirageasuka** [ppy/osu#23458] [Q&A] [answered]
I have encounter this weird situation, after I save progress of the map by press CTRL+S in the editor, it crashed itself instantly with no error messages. But when I boot it back up it turns out that .osu file data has been cleared.

I don't know what really happen and still confuse me for a bit. I have attach event viewer log.

[eventviewer log.txt](https://github.com/ppy/osu/files/11444535/eventviewer.log.txt)
[runtime.log](https://github.com/ppy/osu/files/11444537/runtime.log)

**Comment by bdach** [comment, marked as answer]
Issues with stable should be reported on https://github.com/ppy/osu-stable-issues/

**Comment by peppy** [comment]
You can find temporary files for beatmaps in your temp folder. https://github.com/ppy/osu-stable-issues/issues/1045 may be of help.
